lib/Doctrine/MongoDB/CursorInterface.php

```diff
@@ -253,4 +253,13 @@ public function tailable($tail = true);
      * @return self
      */
     public function timeout($ms);
+
+    /**
+     * Wrapper method for MongoCursor::maxTimeMS().
+     *
+     * @see http://php.net/manual/en/mongocursor.maxtimems.php
+     * @param integer $ms
+     * @return self
+     */
+    public function maxTimeMS($ms);
```

Comment:
Technically speaking, yes. This is a BC break and thus would force us to go 2.0. I'm kicking myself now for not catching all methods that were added in the 1.5 driver.